### PR TITLE
minor optimize: reduce growth and gc

### PR DESCRIPTION
**Pull request by wulianglongrd**

### CODE REVIEW

File: pilot/pkg/model/authorization.go [hunk 1]
```diff
@@ -50,13 +50,22 @@ func GetAuthorizationPolicies(env *Environment) *AuthorizationPolicies {
 
 	policies := env.List(gvk.AuthorizationPolicy, NamespaceAll)
 	sortConfigByCreationTime(policies)
+
+	stats := make(map[string]int)
```
Comment (howardjohn) [inline pilot/pkg/model/authorization.go:54]: nit: `stats` is a somewhat odd name. Maybe `policyCount` or something?

**Review by howardjohn — COMMENTED** [review]
LGTM. thanks

File: pilot/pkg/model/authorization.go (hunk 1 above)
Comment (wulianglongrd) [reply pilot/pkg/model/authorization.go]: thanks, done.

**Review by howardjohn — APPROVED** [review]

**Pull request merged**